docs/index.md

```diff
@@ -28,6 +28,7 @@ The database has some subject-area-specific schemas as well as a default public
 * [Nursery schema](schema/all/nursery/relationships.html) (`nursery`)
 * [Seed bank schema](schema/all/seedbank/relationships.html) (`seedbank`)
 * [Tracking schema](schema/all/tracking/relationships.html) (`tracking`)
+* Document Producer schema (`documentproducer`)
```

Comment:
What do you think of using a shorter name for this schema? Maybe `docprod`, though the specific name isn't important.

I'm thinking about when we're writing ad-hoc SQL queries to debug stuff; a long name is potentially going to be kind of tiresome to have to type out each time you reference a table in this schema.

Reply:
Sounds good, `docprod` it is!